docs/pre.sh

```diff
@@ -3,40 +3,40 @@
 ## Create modules pages
 for D in ../x/*; do
   if [ -d "${D}" ]; then
-    MODDOC=docs/modules/$(echo $D | awk -F/ '{print $NF}')
+    MODDOC=docs/build/modules/$(basename "$D")
     rm -rf $MODDOC
-    mkdir -p $MODDOC && cp -r $D/README.md "$_"
+    mkdir -p $MODDOC && cp -r $D/README.md "$MODDOC/$(basename "$D").md"
```

Comment:
I still do not understand why they cannot stay as README.md.
Docusaurus will render those properly and this avoids the link breakage you have in modules docs.

Reply:
Hmm, okay true you. The reason I added this in was bc it was reading as README.md when i built. Ill remove those then